File: Leet_Code-412/Leet_Code-412.java
/*
 * Question 412
 */

 import java.util.ArrayList;
import java.util.List;
 class Solution {
     public List<String> fizzBuzz(int n) {
         List<String> answer = new ArrayList<>();
         for (int x = 1; x < n+1; x++){
             if (x%3==0 && x%5==0){
                 answer.add("FizzBuzz");
             } else if (x%3==0){
                 answer.add("Fizz");
             } else if (x%5==0){
                 answer.add("Buzz");
             } else {
                 answer.add(Integer.toString(x));
             };
         };
         return answer;
     }
 }

File: Leet_Code-412/Leet_Code-412.py
'''
Question 412
'''

class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer = []
        for x in range(1,n+1):
            if x % 3 == 0 and x % 5 == 0:
                answer.append('FizzBuzz')
            elif x % 3 == 0:
                answer.append('Fizz')
            elif x % 5 == 0:
                answer.append('Buzz')
            else:
                answer.append(str(x))
        return answer